File: 3st/Jaeyoung/인덱스 바꾸기.js
function solution(my_string, num1, num2) {
    var answer = '';
    for( var i = 0 ; i < my_string.length ; i++ ) {
        if( i == num1 ) answer += my_string[num2]
        else if( i == num2 ) answer += my_string[num1]
        else answer += my_string[i]
    }
    return answer;
}

File: 4st/Eunseong/인덱스 바꾸기.java
class Solution {
    public String solution(String my_string, int num1, int num2) {
        String answer = "";
        
        String str[] = my_string.split("");
        String temp = str[num1];
        str[num1] = str[num2];
        str[num2] = temp;
        
        for (int i = 0; i < str.length; i++) {
            answer += str[i];
        }
        
        return answer;
    }
}